syntax = "proto3";

service WorkerService {
    rpc AssignWork(AssignWorkRequest) returns (AssignWorkReply) {}
}

message AssignWorkRequest {
    string path = 1;
    string mode = 2;
    string class = 3;
    string file = 4;
    string task_uuid = 5;
}

message AssignWorkReply {
    bool ok = 1;
}
